### ISSUE TITLE
material-expansionpanel-set seems not to work under a material-tab

### ISSUE THREAD

**Issue by PeterWillems** [dart-lang/angular2_components#54]
The problem is rather simple, see the template below:

```
<material-tab-panel>
  <material-tab label="test">
    <material-expansionpanel-set>
      <material-expansionpanel name="panel 1">
        Test 1
      </material-expansionpanel>
      <material-expansionpanel name="panel 2">
        Test 2
      </material-expansionpanel>
    </material-expansionpanel-set>
  </material-tab>
</material-tab-panel>
```
The Dartium browser renders nothing. If I remove the surrounding material-tab-panel and material-tab the expansion-panels are rendered with the expected accordion behavior. In reverse without the expensionpanel-set the tab is rendered correctly. It seems that the combination causes the problem. Compiling to JavaScript does not change this outcome.

**Comment by nshahan** [comment]
Thanks for reporting. This sounds like it is caused by the same issue reported here: #46.

We have landed a fix internally that should be in the next release.

**Comment by PeterWillems** [comment]
Quick response (for the weekend :-)) thank you. I will wait for the next release.

**Comment by nshahan** [comment]
A fix for the repro I had landed in #57.
